Controllers/ExpenseController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;

```

Comment:
Per VisualStudio man rodo kad yra nenaudojamų šitų using, pilkai pažymėti. Tau nerodo? Juos galima išmest, nes jų niekam nereikia.